src/test/java/com/synopsys/integration/blackduck/comprehensive/NotificationsTestIT.java

```diff
@@ -100,6 +96,10 @@ public void testProjectNotifications() throws IntegrationException, InterruptedE
         }
 
         assertEquals(expectedKeys, foundKeys);
+
+        // CLEAN UP
+        blackDuckApiClient.delete(projectVersionWrapper2.getProjectVersionView());
+        blackDuckApiClient.delete(projectVersionWrapper.getProjectView());
```

Comment:
@DanaMaxfield We had agreed on replacing the deletes with an update notification, but after lots of trial I could not get that to happen (explanation below). So I've decided to just have this test confirm two CREATE notifications are created. Then at the very end, we do the delete (by which point we would hope the soft created project versions have been updated in the database already). 


I don't know if an "UPDATE" notification is ever created for any reason. Though ProjectNotificationContent has three types of possible OperationTypes: 
CREATE
DELETE
UPDATE

Doing an update to a project did not yield an "... UPDATE ....' notification. I also did not find a notification like that when looking at examples on a couple of test instances: 

<img width="916" alt="Screenshot 2024-09-05 at 3 23 45 PM" src="https://github.com/user-attachments/assets/9f589b2e-af9a-468f-8f94-9a1ab8e073fc">

For example I tried updating different project fields (any that did not require lots more state set up like creating user groups, policies, etc). I would get a log entry like saying an update was performed, and indeed I would see the change in the UI but not "UPDATE" notification would be created.

Reply:
I don't believe an update notification to a project is possible.  A project is created or deleted and updates come from BomEditNotifications which aren't displayed in the UI.  They are a notification type that Alert uses and you need to filter for that notification type when using the REST API calls.

Reply:
Operation types are the types of REST API calls you can make against the resource.  You can create a project notification, update a project notification i.e. mark it as viewed, and delete the notification.